app/build.gradle.kts

```diff
@@ -0,0 +1,147 @@
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+import com.github.spotbugs.snom.SpotBugsTask
+
+plugins {
+    id("com.android.application")
+    id("com.github.spotbugs")
+}
+
+spotbugs {
+    ignoreFailures.set(false)
+    setEffort("max")
+    excludeFilter.set(file("./config/spotbugs/exclude.xml"))
+    reportsDir.set(file("$buildDir/reports/spotbugs/"))
+}
+
+android {
+    namespace = "protect.card_locker"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "me.hackerchick.catima"
+        minSdk = 21
+        targetSdk = 33
+        versionCode = 131
+        versionName = "2.26.0"
+
+        vectorDrawables.useSupportLibrary = true
+        multiDexEnabled = true
+
+        resourceConfigurations += listOf("ar", "bg", "bn", "bn-rIN", "bs", "cs", "da", "de", "el-rGR", "en", "eo", "es", "es-rAR", "fi", "fr", "he-rIL", "hi", "hr", "hu", "in-rID", "is", "it", "ja", "ko", "lt", "lv", "nb-rNO", "nl", "oc", "pl", "pt", "ro-rRO", "ru", "sk", "sl", "sv", "tr", "uk", "zh-rTW", "zh-rCN")
```

Comment:
This will break the `locales.py` script, that script will also need updating (but should be a very minor change): https://github.com/CatimaLoyalty/Android/blob/146a4676d5ba8f2685acb857642c6499cbf518c7/.scripts/locales.py#L22

Reply:
This should be fixed now